skills/managing-google-workspace/references/apps-script.md

```
@@ -0,0 +1,182 @@
+
+| Parameter | Type | Required | Default | Notes |
+|-----------|------|----------|---------|-------|
+| user_google_email | string | yes | | |
+| script_id | string | yes | | |
+| file_name | string | yes | | |
+
+### update_script_content
+Updates or creates files in a script project.
+
+| Parameter | Type | Required | Default | Notes |
+|-----------|------|----------|---------|-------|
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)